src/components/utils/rhf-inputs/directory-items-input.tsx

```diff
@@ -76,12 +82,23 @@ const DirectoryItemsInput: FunctionComponent<DirectoryItemsInputProps> = ({
 
     const addElements = useCallback(
         (values: any[]) => {
+            const allChips = getValues(name);
+
+            // if we select a chip and return a new values, we remove it to be replaced
+            if (selectedChip && selectedChip?.length > 0 && values.length > 0) {
+                selectedChip.forEach((chip) => {
+                    remove(
+                        allChips.findIndex((item: any) => item.id === chip.id)
+                    );
+                });
+            }
+
             values.forEach((value) => {
                 const { icon, children, ...otherElementAttributes } = value;
 
                 // Check if the element is already present
                 if (
-                    getValues(name).find(
+                    allChips.find(
```

Comment:
If we select several elements while keeping the element we clicked on, we will get the error message 'This element is already used'. Is this intended?